pkgs/development/python-modules/pyspice/default.nix

```
@@ -0,0 +1,51 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, libngspice
+, numpy
+, ply
+, scipy
+, pyyaml
+, cffi
+, requests
+, matplotlib
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "PySpice";
+  version = "1.4.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1j7si4jrrdf13070dcl4qr19rpxnnlqq4ygnqgzpq4whdcfp398q";
+  };
+
+  buildInputs = [
```

python packages need to go on propagatedBuildInputs
```suggestion
  propagatedBuildInputs = [
```